src/wp-includes/pluggable.php

```diff
@@ -1068,10 +1068,12 @@ function wp_set_auth_cookie( $user_id, $remember = false, $secure = '', $token =
 		 * Allows preventing auth cookies from actually being sent to the client.
 		 *
 		 * @since 4.7.4
+		 * @since 6.2.0 The $user_id parameter was added.
```

Comment:
For the consistency.

```suggestion
		 * @since 6.2.0 The `$user_id` parameter was added.
```